src/backend/renderer/element/solid.rs

```diff
@@ -251,10 +253,10 @@ impl SolidColorRenderElement {
     ) -> Self {
         let geo = Rectangle::from_loc_and_size(location, buffer.size.to_physical_precise_round(scale));
         let color = [
-            buffer.color[0] * alpha,
-            buffer.color[1] * alpha,
-            buffer.color[2] * alpha,
-            buffer.color[3] * alpha,
+            buffer.color.r() * alpha,
+            buffer.color.g() * alpha,
+            buffer.color.b() * alpha,
+            buffer.color.a() * alpha,
```

Comment:
Nitpick: Couldn't multiplying by an alpha be a method of `Color32F`, so we can then use `components`?

Reply:
Sure, I thought about adding a `Mul` impl or do you prefer some explicit function for that?
(No need to use components, `draw_solid` also takes `Color32F`)

Reply:
I think `Mul` is good enough.

Reply:
done